client/reader/stream/post-lifecycle.jsx

```diff
@@ -51,6 +54,39 @@ export default class PostLifecycle extends React.PureComponent {
 		this.updatePost( nextProps );
 	}
 
+	shouldComponentUpdate( nextProps, nextState ) {
+		const {
+			isSelected, suppressSiteNameLink, showPostHeader, showFollowInHeader,
+			showPrimaryFollowButtonOnCards, showSiteName, isDiscoverStream,
+		} = this.props;
+
+		const shouldUpdate = this.state.post !== nextState.post ||
```

Comment:
its not the full set.  I don't like the idea of skipping some though because that will cause bugs in the future.

Currently checking if shallow compare would have the same effect